vms/platformvm/vm.go

```diff
@@ -90,8 +90,11 @@ type VM struct {
 	txBuilder txbuilder.Builder
 	manager   blockexecutor.Manager
 
-	startShutdown context.CancelFunc
-	awaitShutdown sync.WaitGroup
+	// Cancelled on shutdown
+	onShutdownCtx context.Context
```

Comment:
This technically is not needed in the `VM` struct. It feels more readable to have it here though